next.config.js

```diff
@@ -1,13 +1,15 @@
 const path = require("path"); // eslint-disable-line @typescript-eslint/no-var-requires
+// for custom styles in SASS
+const withSass = require('@zeit/next-sass');
 
 const BASE_PATHNAME = "events";
 
-module.exports = {
+module.exports = withSass({
   sassOptions: {
     includePaths: [path.join(__dirname)],
   },
   assetPrefix: BASE_PATHNAME ? `/${BASE_PATHNAME}` : "",
   publicRuntimeConfig: {
     basePath: BASE_PATHNAME ? `/${BASE_PATHNAME}` : "",
   },
-};
+});
```

Comment:
It seems that Next.JS has sass support out of the box (or at least has it with the sass node module). Why do we need withSass?